src/main/java/org/example/homework_nr_9/MeetingRoom.java

```diff
@@ -0,0 +1,36 @@
+package org.example.homework_nr_9;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class MeetingRoom {
+    List<AttenTraining> inputlist;
+    public MeetingRoom(ArrayList<AttenTraining> train) {
```

Comment:
Change here from ArrayList to List<AttendTraining>. that way you won't need to cast lists